src/SelectField/SelectField.js

```diff
@@ -180,6 +185,7 @@ class SelectField extends Component {
       <TextField
         {...other}
         style={style}
+        defaultText={hintText || floatingLabelText ? '' : defaultText}
```

Comment:
Well, for all the cases I have, there's a default action when nothing is selected, but I don't want to keep the select box empty. A simple solution would be to specify the text from the first option here as well, as other people still want this select empty for their cases.